CONTRIBUTING.rst

```diff
@@ -59,3 +59,15 @@ To (re-)build the HTML files, use::
    python3 setup.py build_sphinx
 
 The generated files will be available in the directory ``build/sphinx/html/``.
+
+
+Auto-Generated Documentation for Pull Requests
+----------------------------------------------
+
+Whenever a Pull Request (PR) is created on
+https://github.com/mgeier/insipid-sphinx-theme/pull/,
+the documentation is automatically built on https://readthedocs.org/.
+On the PR page, click on the green check mark
+(or on the red "x" in case of a test failure),
```

Comment:
This refers to single commits. Maybe it's easier to understand to point to the check summary shown at the bottom of the PR, e.g.
![image](https://user-images.githubusercontent.com/173624/163963326-893bda57-2af5-4914-ba63-581500077ab6.png)
?

Reply:
Yeah, that's what I wanted to write first, but then I saw that the summary is only shown when logged into Github.

I guess most potential contributors will be logged in anyway, but who knows.

> This refers to single commits.

Yes, but all of them should lead to the same HTML page, since the URL is always the same.

Reply:
Ah, ok, wasn't aware that you need to be logged in.

So maybe be more explicit, e.g.

```rst
On the PR page,
click on the green or red check mark
of a single commit,
look for the **readthedocs.org** entry and click on "Details"
to see the auto-generated HTML pages.
```

But I'm also fine with the current solution.

Reply:
I've re-based this to get rid of the build error. I've also mentioned the "latest commit" (even though it doesn't really have to be the latest one).

I hope that's clearer now.

Reply:
Yes, all fine from my side.